app-modules/assist-data-model/src/Providers/AssistDataModelServiceProvider.php

```diff
@@ -20,5 +22,12 @@ public function boot(): void
         Relation::morphMap([
             'student' => Student::class,
         ]);
+
+        $this->registerObservers();
+    }
+
+    protected function registerObservers(): void
+    {
+        Caseload::observe(CaseloadObserver::class);
```

Comment:
Shouldn't this be in the CaseLoadManagementServiceProvider?

Reply:
Whoops!

Reply:
Moved